src/query/storages/table-meta/src/meta/v1/snapshot.rs

```diff
@@ -68,6 +70,7 @@ impl TableSnapshot {
         summary: Statistics,
         segments: Vec<Location>,
         cluster_key_meta: Option<ClusterKey>,
+        table_statistics_location: Option<String>,
```

Comment:
cc @dantengsky   Do we need this? Maybe we could store this using a fixed location by table id?

Reply:
For the evolution of the table statistics file format, I suggest keeping the location specified explicitly (if I get this pr correct, the version of the statistics file is detected by parsing its location)